tests/cross/src/test/scala/tests/pc/CompletionWorkspaceSuite.scala

```diff
@@ -334,7 +334,7 @@ class CompletionWorkspaceSuite extends BaseCompletionSuite {
   )
 
   checkEdit(
-    "match-typed",
+    "match-typed".tag(IgnoreScala3),
```

Comment:
```suggestion
    "match-typed",
```
this shouldn't be ignored without a reason.

Reply:
So shouldn't we just add a comment that there are no completions for null instead of removing tag?